blog/_posts/2025-04-03-Scaling Kubernetes -for-AI-and-Data-intensive -Workloads

```diff
@@ -0,0 +1,229 @@
+---
+title: "Scaling Kubernetes for AI and Data-intensive Workloads: The AKS Fleet Strategy
+"
+description: "Using Multi-cluster approach to achieving high compute scale on K8s, why , and how to do it with Azure Kubernetes Fleet Manager"
+date: 2025-04-03
```

Comment:
This date is set to 4/3 which means it won't publish until Thursday. Change to 4/2 or earlier if that matters to you